Makefile

```diff
@@ -21,16 +23,30 @@ golangci-lint: ## Run golangci-lint on codebase
 	@echo Running golangci-lint
 	golangci-lint run ./...
 
+.PHONY: test
 test:
 	go test -v -mod=readonly -failfast -race ./...
 
+.PHONY: build
 build:
 	mkdir -p dist
 	env GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o dist/rtcd -ldflags '$(LDFLAGS)' -mod=readonly -trimpath ./cmd/rtcd
 
+.PHONY: docker-build
+docker-build:
+	@if ! [ -x "$$(command -v docker)" ]; then \
+		echo "Docker is not installed. Please see https://docs.docker.com/get-docker for installation instructions."; \
+		exit 127; \
+	fi; \
+
+	@[ "${tag}" ] || ( echo "build-docker requires a tag. Set a tag with: \"make docker-build tag=rtcd:my_tag\""; exit 1 )
+
+	docker build -f build/Dockerfile . -t $(tag)
```

Comment:
nice, not for now but in a later commit let's make [https://github.com/mattermost/rtcd/blob/master/build/Dockerfile#L4] an `ARG` so we can change this easily with `--build-arg`